core/services/keeper/upkeep_executer.go

```diff
@@ -316,11 +329,15 @@ func buildJobSpec(
 ) map[string]interface{} {
 	return map[string]interface{}{
 		"jobSpec": map[string]interface{}{
-			"jobID":                 jb.ID,
-			"fromAddress":           upkeep.Registry.FromAddress.String(),
-			"contractAddress":       upkeep.Registry.ContractAddress.String(),
-			"upkeepID":              upkeep.UpkeepID.String(),
-			"prettyID":              upkeep.PrettyID(),
+			"jobID":           jb.ID,
+			"fromAddress":     upkeep.Registry.FromAddress.String(),
+			"senderAddress":   senderAddress.String(),
+			"contractAddress": upkeep.Registry.ContractAddress.String(),
+			"upkeepID":        upkeep.UpkeepID.String(),
+			"prettyID":        upkeep.PrettyID(),
+			"pipelineSpec": &pipeline.Spec{
+				ForwardingAllowed: jb.ForwardingAllowed,
+			},
```

Comment:
curious, where is this used? I don't see it anywhere in observation source

Reply:
this is used directly in eth_tx